### DISCUSSION TITLE
What does cleanupInterval do

### DISCUSSION THREAD

**Discussion by subhradeepnag** [arthurfiorette/axios-cache-interceptor#861] [General]
Hi,

I was reviewing the documentation for axios-cache-interceptor and noticed a property called cleanupInterval. According to the documentation, it says the _The interval in milliseconds to run a setInterval job of cleaning old entries. If false, the job will not be created. Disabled is default_

Could you clarify if this means the entire cache is cleared after the specified interval, or if it specifically targets certain old entries? I'm particularly confused about what is meant by "old entries."

Thanks,

Subhradeep

**Comment by dgomez404** [comment]
Hello @subhradeepnag, I'm looking how to delete old entries. Did you find out how to delete them? 

I tried with cleanupInterval, but it does not work.

**Comment by arthurfiorette** [comment]
https://github.com/arthurfiorette/axios-cache-interceptor/blob/f20a9a692cb7f56ae17a36de1363c59f1d0fb731/src/storage/memory.ts#L102
